.github/pull_request_template.md

```diff
@@ -17,22 +17,14 @@ being proposed in this PR (i.e. feat, test, fix, ci, chore, docs).
 <!-- This change MUST also be proposed to all newer, and still supported,
 releases. List the corresponding PRs, or ignore if not applicable. -->
 
-## Testing
-<!-- Provide proof of testing and/or testing instructions, when applicable,
-to help speed up the review process. -->
-
-```bash
-# Example:
-#  chisel cut ... --root <path> <pkg>_<proposed-slice>
-#  sudo chroot <path> <app>
-```
 
 ## Checklist
 <!-- Go over all the following points, and put an `x` in all the boxes
 that apply. -->
 
 * [ ] I have read the [contributing guidelines](
 https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
+* [ ] I have tested my changes
```

Comment:
```suggestion
* [ ] I have tested my changes ([How to](https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md))
```

Reply:
Hey, I see you edited the suggestion but didn't apply them.

Reply:
dunno what happened. committed one by one now